src/App.js

```diff
@@ -5,12 +5,9 @@ import './index.css';
 class App extends React.PureComponent {
   render() {
     return (
-      <div className="App">
-        <div className="App-container">
-          <Calculator />
-        </div>
-      </div>
+      <Calculator />
     );
   }
```

Comment:
Thanks for the review @BenMukebo 
But this is the requirement Refactor the Calculator component **(components/Calculator.js)** from a class-based component to a functional component with hooks, not the app component.
And trying that which you suggested is not working

Reply:
Hi @Christabelle, you're right, it wasn't in the project requirements, but it's also important to refactor the app component since you're learning how to use React [hooks] (https://reactjs.org/docs /hooks-intro.html).